tokens/transformation/styledComponents/formatStyledComponentsTheme.js

```diff
@@ -0,0 +1,55 @@
+const camelCase = require('lodash.camelcase')
+const fileHeader = require('../web/fileHeader')
+
+function isToken (tokenOrTokenCategory) {
+  return !!tokenOrTokenCategory.type
+}
+
+function cleanKey (key) {
+  return camelCase(key.trim())
+}
+
+function deThemeColorTokens(tokenCategory) {
+  const result = {}
+  for (const key in tokenCategory) {
+    const value = tokenCategory[key]
+    if (isToken(value)) {
+      result[key] = {
+        ...tokenCategory[key],
+        name: tokenCategory[key].name?.replace('-light-', '-')
```

Comment:
Just to confirm I'm understanding this right, we only pass in the light tokens here to get an unthemed version of the colors?

Reply:
Yeah it's weird so I'll at least add a comment